PebbleSharp.Core/Apps/AppFetchResponsePacket.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using PebbleSharp.Core.Responses;
+namespace PebbleSharp.Core
+{
+	[Endpoint(Endpoint.AppFetch)]
+	public class AppFetchResponsePacket
+	{
+		public byte Command { get; set; }
+		public AppFetchStatus Response { get; set; }
+
+		public AppFetchResponsePacket()
+		{
+			Command = 1;
+		}
+
+		public byte[] GetBytes()
+		{
+			var bytes = new List<byte>();
+			bytes.Add(Command);
+			bytes.Add((byte)Response);
+			return bytes.ToArray();
+		}
```

Comment:
Though I like the use of List<byte> when building the array is more complex for simple array I favor using the array initializer to avoid the creation of a List object that is simply discarded.